src/token-processor/queue/job/process-smart-contract-job.ts

```diff
@@ -63,6 +73,30 @@ export class ProcessSmartContractJob extends Job {
     return await client.readUIntFromContract('get-last-token-id');
   }
 
+  private async enqueueSftContractTokenIds(contract: DbSmartContract): Promise<void> {
+    const tokenInserts: DbTokenInsert[] = [];
+    // Scan for `sft-mint` events emitted by the SFT contract.
+    const cursor = this.apiDb.getSmartContractLogsByContractCursor({
+      contractId: contract.principal,
+    });
+    for await (const rows of cursor) {
+      for (const row of rows) {
+        const event = getContractLogSftMintEvent(row);
+        if (!event) {
+          continue;
+        }
+        tokenInserts.push({
+          smart_contract_id: contract.id,
+          type: DbTokenType.sft,
+          token_number: Number(event.tokenId), // Not necessarily sequential.
```

Comment:
Can `tokenId`  be any valid Clarity uint (128-bit) value? If so, the pg column type would have to be `numeric` and the js type would have to be `bigint` or string-quoted integer.